src/main/resources/jenkins/plugins/publish_over_ssh/BapSshHostConfiguration/config_no_BV.properties

```diff
@@ -27,6 +27,7 @@ password=P*s*p*r*s* / P*s*w*r*
 keyPath=P*t* t* k*y
 key=K*y
 disableExec=D*s*b*e e*e*
+jumpHost=j*m* *o*t
```

Comment:
It's a localization file that translates things from the config.jelly to different languages. You don't need to worry about that file.